MapboxNavigation/RouteVoiceController.swift

```diff
@@ -140,39 +137,18 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate, AVAudioP
     }
     
     func unDuckAudio() throws {
-        if !speechSynth.isSpeaking {
-            try AVAudioSession.sharedInstance().setActive(false, with: [.notifyOthersOnDeactivation])
-        }
+        try AVAudioSession.sharedInstance().setActive(false, with: [.notifyOthersOnDeactivation])
     }
     
     open func didPassSpokenInstructionPoint(notification: NSNotification) {
-        guard shouldSpeak(for: notification) == true else { return }
-        
-        speak(fallbackInstruction)
-    }
-    
-    func shouldSpeak(for notification: NSNotification) -> Bool {
-        guard isEnabled, volume > 0, !NavigationSettings.shared.muted else { return false }
```

Comment:
It would’ve been fine to keep `shouldSpeak(for:)` around, but in any case, make sure these `guard` statements are in `RouteVoiceController.didPassSpokenInstructionPoint(_:)`, _not_ `PollyVoiceController.didPassSpokenInstructionPoint(_:)` – the latter method should go away.